hw3/01-characters.js

```diff
@@ -1,4 +1,75 @@
-// url for the Thrones API
-const url = 'https://thronesapi.com/api/v2/Characters';
+const url = "https://thronesapi.com/api/v2/Characters";
```

Comment:
AirBnb guide recommends single quotes in JS files. Looks like it may have just been automatically changed by formatter, I have run into my VS Code formatter trying to switch these kinds of lines to double quotes.

Reply:
Yes, that's something I need to remember!